webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/DSRemoteWebknossosClient.scala

```diff
@@ -67,6 +68,16 @@ class DSRemoteWebknossosClient @Inject()(
       .addQueryString("key" -> dataStoreKey)
       .put(dataSource)
 
+  def getOngoingUploadsForUser(userTokenOpt: Option[String], organizationName: String): Fox[List[OngoingUpload]] =
+    for {
+      userToken <- option2Fox(userTokenOpt) ?~> "reserveUpload.noUserToken"
+      datasetIds <- rpc(s"$webknossosUri/api/datastores/$dataStoreName/getOngoingUploadsForUser")
+        .addQueryString("key" -> dataStoreKey)
+        .addQueryString("token" -> userToken)
+        .addQueryString("organizationName" -> organizationName)
+        .getWithJsonResponse[List[OngoingUpload]]
+    } yield datasetIds
```

Comment:
```suggestion
    } yield ongoingUploads
```
(is what is returned here, right? I’d rename the variable accordingly)